<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dgfConfig</class>
 <widget class="QDialog" name="dgfConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>731</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="listBox">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>List of Detectors</string>
      </property>
      <widget class="QListView" name="listView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="confBox">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>350</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>560</width>
          <height>500</height>
         </size>
        </property>
        <property name="title">
         <string>Configuration</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>531</width>
           <height>311</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Polarity</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="polarity">
              <item>
               <property name="text">
                <string>negative</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>positive</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Tau [us]</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="tau">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>999.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Gain [V/V]</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="gain">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="singleStep">
               <double>0.001000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Offset [Volts]</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="offset">
              <property name="decimals">
               <number>4</number>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Slow Length [us]</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="slowLength"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Slow Gap [us]</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QDoubleSpinBox" name="slowGap"/>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Baseline averaging</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QSpinBox" name="baselineAveraging"/>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Fast Length [ticks]</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QSpinBox" name="fastLength"/>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Fast Gap [ticks]</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QSpinBox" name="fastGap"/>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Threshold [ADC units]</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QSpinBox" name="threshold"/>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Use GFLT for gate</string>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QComboBox" name="useGfltForGate">
              <item>
               <property name="text">
                <string>No</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yes</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Baseline cut</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="baselineCut"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Channel gate required</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="channelGateRequired">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>No</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yes</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>GFLT polarity</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="gfltPolarity">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>negative</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>positive</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Gate acceptance polarity</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="gateAcceptancePolarity">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>negative</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>positive</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Gate edge polarity inversion </string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="gateEdgePolarityInversion">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>No</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yes</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Gate window [ticks]</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QSpinBox" name="gateWindow"/>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Gate delaywindow [ticks]</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QSpinBox" name="gateDelayWindow"/>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Pause pile-up rejector</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QComboBox" name="pausePileUpRejector">
              <item>
               <property name="text">
                <string>No</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yes</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Disable pile-up rejector</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QComboBox" name="disablePileUpRejector">
              <item>
               <property name="text">
                <string>No</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yes</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="psaButton">
          <property name="text">
           <string>PSA Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveAllButton">
          <property name="text">
           <string>Save All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
